### PR TITLE
Add Tianyi to team.yml

### PR DESCRIPTION
**Pull request by Tian1Pan**

### CODE REVIEW

**Comment by Tian1Pan** [comment]
@jhodrien I've done it as requested.

**Review by jhodrien — COMMENTED** [review]
Take a look at this previewed locally, as your image is squashed when it renders.

File: _data/team.yml [hunk 1]
```diff
@@ -103,23 +100,22 @@ main:
     shell scripting. Andy has expertise in software sustainability, sensitive data 
     management and computational reproducibility.
     "
-    img: team/andyturner.jpg
-    
+    img: team/andyturner.jpg
```
Comment (jhodrien) [inline _data/team.yml:103]:
```suggestion
    img: team/andyturner.jpg
```

**Review by jhodrien — APPROVED** [review]
Looks good to me.

**Pull request merged**